src/main/java/blackjack/util/BlackjackScoreCalculator.java

```diff
@@ -0,0 +1,32 @@
+package blackjack.util;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Deck;
+
+public class BlackjackScoreCalculator implements ScoreCalculator {
+
+    private static final int ACE_EXCEPTION_NUMBER = 10;
+    private static final int WINNING_NUMBER = 21;
+
+    @Override
+    public int apply(Deck deck) {
+        return getScoreIncludeAce(deck, getScoreExceptAce(deck));
+    }
+
+    private int getScoreExceptAce(Deck deck) {
+        return deck.getCards().stream().filter(card -> !card.isAce()).mapToInt(Card::getScore)
+            .sum();
```

Comment:
이렇게 작성해주시는 것이 가독성이 더 좋아보여요 :)
```suggestion
        return deck.getCards().stream()
                .filter(card -> !card.isAce())
                .mapToInt(Card::getScore)
                .sum();
```

그리고 deck에서 Cards를 꺼내지 말고 Deck이 계산된 점수를 주도록 바꿔보면 좋을 것 같아요.

Reply:
deck에서 총 점수와 ace의 갯수만 가져와 추가적인 계산은 calculator에서 작업하는 방식으로 바꾸었습니다!